### ISSUE TITLE
model.py 的429行，在Fine-tuning on CMU-MOSI的时候报错

### ISSUE THREAD

**Issue by wupeng1989** [zrguo/MPLMM#2]
你好，很感谢你的工作。
先Pre-train the model on CMU-MOSEI without prompts 后，然后执行Fine-tuning on CMU-MOSI的代码时候，在model.py的429行， h_ls = torch.cat([h_ls, batch_prompt[0]], dim=0)，提示报错Sizes of tensors must match except in dimension 0.
h_ls的维度是:[50,8,60], batch_prompt[0]的维度是[8,16,60]，这个变量cat会提示报错。
期待你的回复，谢谢！

**Comment by supercool706** [comment]
我也是遇到了这个问题TAT

**Comment by zrguo** [comment]
你好，代码整理时出现纰漏，现已更新。

**Comment by version98** [comment]
> 是[8,16,

Hello, did you perform feature extraction on your dataset yourself? And are they still undergoing training?
您好，请问您复现的时候，是自己重新提取了数据集特征吗？

**Comment by czy-a** [comment]
> > 是[8,16,
> 
> Hello, did you perform feature extraction on your dataset yourself? And are they still undergoing training? 您好，请问您复现的时候，是自己重新提取了数据集特征吗？

你好，想问一下这个数据集是需要根据缺失的mode来对原始完整数据进行缺失处理吗

**Comment by zrguo** [comment]
> 请问一下用到的这几个数据集
> 
> > 你好，代码整理时出现纰漏，现已更新。
> 
> 你好，请问用到的数据集是在哪里获取的吗，好像格式和用到的不一样

论文中特征提取部分所引用论文对应的仓库下提供了提取好的特征，可以直接下载，比如，MOSEI和MOSI可在[mult](https://github.com/yaohungt/Multimodal-Transformer)仓库下获取。其余也可在对应论文的仓库下下载。另外，使用任何提取特征的方法都可以，只要保证dataset中有相应的获取特征维度和missing code函数即可。

**Comment by TianBaoGe** [comment]
> > 请问一下用到的这几个数据集
> > > 你好，代码整理时出现纰漏，现已更新。
> > 
> > 
> > 你好，请问用到的数据集是在哪里获取的吗，好像格式和用到的不一样
> 
> 论文中特征提取部分所引用论文对应的仓库下提供了提取好的特征，可以直接下载，比如，MOSEI和MOSI可在[mult](https://github.com/yaohungt/Multimodal-Transformer)仓库下获取。其余也可在对应论文的仓库下下载。另外，使用任何提取特征的方法都可以，只要保证dataset中有相应的获取特征维度和missing code函数即可。

谢谢作者，还有个问题：测试代码和训练代码貌似都是随机抽取模态（0～5），我理解测试是不是测六次，每次都固定missing_mode，然后取六次结果平均值

**Comment by zrguo** [comment]
> > > 请问一下用到的这几个数据集
> > > > 你好，代码整理时出现纰漏，现已更新。
> > > 
> > > 
> > > 你好，请问用到的数据集是在哪里获取的吗，好像格式和用到的不一样
> > 
> > 
> > 论文中特征提取部分所引用论文对应的仓库下提供了提取好的特征，可以直接下载，比如，MOSEI和MOSI可在[mult](https://github.com/yaohungt/Multimodal-Transformer)仓库下获取。其余也可在对应论文的仓库下下载。另外，使用任何提取特征的方法都可以，只要保证dataset中有相应的获取特征维度和missing code函数即可。
> 
> 谢谢作者，还有个问题：测试代码和训练代码貌似都是随机抽取模态（0～5），我理解测试是不是测六次，每次都固定missing_mode，然后取六次结果平均值

可以这么做，代码中是像论文fig4中测试在不同缺失率下模态随机缺失的性能。

**Comment by ShilinSun** [comment]
> > > > 请问一下用到的这几个数据集
> > > > > 你好，代码整理时出现纰漏，现已更新。
> > > > 
> > > > 
> > > > 你好，请问用到的数据集是在哪里获取的吗，好像格式和用到的不一样
> > > 
> > > 
> > > 论文中特征提取部分所引用论文对应的仓库下提供了提取好的特征，可以直接下载，比如，MOSEI和MOSI可在[mult](https://github.com/yaohungt/Multimodal-Transformer)仓库下获取。其余也可在对应论文的仓库下下载。另外，使用任何提取特征的方法都可以，只要保证dataset中有相应的获取特征维度和missing code函数即可。
> > 
> > 
> > 谢谢作者，还有个问题：测试代码和训练代码貌似都是随机抽取模态（0～5），我理解测试是不是测六次，每次都固定missing_mode，然后取六次结果平均值
> 
> 可以这么做，代码中是像论文fig4中测试在不同缺失率下模态随机缺失的性能。

你好，请问，Table 1中的缺失情况仅仅指的是测试集缺失指定missing code和缺失率70%，非主干网络训练时仍然按照缺失率70%，missing code随机设置么？
